lib/ansible/modules/cloud/opennebula/one_vm.py

```diff
@@ -1300,15 +1300,16 @@ def get_connection_info(module):
     if not username:
         if not password:
             authfile = os.environ.get('ONE_AUTH')
-            if authfile is not None:
-                try:
-                    authstring = open(authfile, "r").read().rstrip()
-                    username = authstring.split(":")[0]
-                    password = authstring.split(":")[1]
-                except BaseException:
-                    module.fail_json(msg="Could not read ONE_AUTH file")
-            else:
-                module.fail_json(msg="No Credentials are set")
+            if authfile is None:
+                authfile = os.path.join(os.environ.get("HOME"), ".one", "one_auth")
+            try:
+                authstring = open(authfile, "r").read().rstrip()
+                username = authstring.split(":")[0]
+                password = authstring.split(":")[1]
+            except (OSError, IOError) as e:
```

Comment:
```suggestion
            except (OSError, IOError):
```
Since you are not using `e`.

Reply:
@Akasurde  can you review again? I made a new commit yesterday.